src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -223,6 +222,67 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
```

Comment:
let's please do the same exercise as brakes and body for vehicle-summary please (see the common fields below in the screenshot).
Then psv vehicle summary should show these additional fields: speed limiter exempt, tacho, euro, fuel, number of wheels driven, emission, seat lower decks, seat upper decks, standing capacity, vehicle size, number of seat belts, seat belt installations


![image](https://user-images.githubusercontent.com/62697030/82083519-c5e2bb00-96e1-11ea-8daf-6e48223670c7.png)